modules/svg/BUILD.gn

```diff
@@ -11,7 +11,7 @@ if (skia_enable_svg) {
     include_dirs = [ "include" ]
   }
 
-  skia_component("svg") {
+  skia_shared_library("svg") {
```

Comment:
The macro changes each lib to use the global setting. If we do a static build and force this lib to be shared we get all modules statically linked into this shared lib